addon/lib/Feature.jsm

```diff
@@ -8,53 +8,108 @@ Cu.import("resource://gre/modules/XPCOMUtils.jsm");
 
 
 /* eslint no-unused-vars: ["error", { "varsIgnorePattern": "(EXPORTED_SYMBOLS|Feature)" }]*/
-const EXPORTED_SYMBOLS = this.EXPORTED_SYMBOLS = ["Feature"];
+const EXPORTED_SYMBOLS = this.EXPORTED_SYMBOLS = ["feature"];
 
+const BASERESOURCE = "perception-study";
 
 XPCOMUtils.defineLazyModuleGetter(this, "RecentWindow",
   "resource:///modules/RecentWindow.jsm");
-
+XPCOMUtils.defineLazyModuleGetter(this, "config",
+  `resource://${BASERESOURCE}/Config.jsm`);
+XPCOMUtils.defineLazyModuleGetter(this, "studyUtils",
+  `resource://${BASERESOURCE}/StudyUtils.jsm`);
 
 class Feature {
-  startup(config) {
-    console.log(`Starting the thing with ${config}`);
-    return "started";
+  constructor() {
+    this.notificationBar = new Notification();
+    this.TREATMENTS = {
```

Comment:
so, this isn't aactually what we want, structurally.  we just want one treatment, but I am not sure here.

Reply:
Well my thinking was TREATMENTS were associated with the feature, so should be in the Feature class. There would only be one TREATMENTS assuming we only create one instance of the Feature. This was just an attempt to minimize the number of global variables. Feel free to move around!